routes/signup.js

```diff
@@ -3,9 +3,9 @@
  * @apiName SignupUser
  * @apiGroup Auth
  *
- * @apiParam {String} display_name User's display name.
- * @apiParam {String} email User's email.
- * @apiParam {String} password User's password.
+ * @apiParam (Request body) {String} display_name User's display name.
```

Comment:
Just noticed it defaults to parameters, i.e. `req.param`; so I used parens to group the params as "Request body", i.e. `req.body`. We'll still want to use just `@apiParam` (without grouping) for whatever params other routes have like `cid` in `/chat/:cid`.